sql/trace.go

```diff
@@ -79,20 +80,35 @@ func (n *explainTraceNode) Next() bool {
 			basePos = n.lastPos + 1
 		}
 
+		// Iterate through once to determine earliest timestamp.
+		var earliest time.Time
+		for _, sp := range n.txn.CollectedSpans {
+			for _, entry := range sp.Logs {
+				if entry.Timestamp.Before(earliest) || earliest.IsZero() {
+					earliest = entry.Timestamp
+					break
```

Comment:
is this right? i think you need to iterate through them all to find the earliest.